### ISSUE TITLE
Old ABS ids

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#1711]
Replaces https://github.com/whosonfirst-data/whosonfirst-data-admin-au/issues/13:

Hi Guys

It looks like many Australian localities have a field called "qs:la_lc" which contains a Australian Bureau of Statistics ID in it.

This ID (Starts with SSC for localities) is extremely useful as it allows you to easily fetch a wide range of Census demographic information for that area, and it's also used in other datasets as a concordance such as WikiData which means it's also useful for linking between datasets.

Unfortunately it looks like "qs:la_lc" is derived from the 2011 Census dataset.
The ABS releases a new set of IDs with each Census, the IDs don't carry across from Census to Census.

E.g. Acacia Gardens has IDs 2011: SSC10010 and 2016: SSC10014.

I can see utility in keeping both IDs and other projects do store both, making a note which Census dataset the ID belongs to.
E.g. https://www.wikidata.org/wiki/Q7966919 has both 2011 and 2016 SSC IDs listed.

Can I propose that we deprecate and remove the "qs:la_lc" property?
And instead store these IDs in the concordances property as "ausstat:2011" and "ausstat:2016"?
This would be very clear, allows us to store both at the same time and is future proof for the 2021 census which will again use different IDs.

**Comment by stepps00** [comment]
per @nickstallman

**Comment by nvkelso** [comment]
Good idea to track each vintage separately, especially if the unique IDs are not stable!

The`qs` properties are there for historic spelunking, we don't recommend them for daily use.

One option is to add a new source for the 2016 Australian census, to complement the existing, generic Australian census source, which is really the 2011 census. 

- https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/ausstat.json

We should note the existing source is the 2011 census, then the new source could be `ausstat_2016`?

**Comment by stepps00** [comment]
Yeah, agree that we should note the 2011 vs 2016 PSMA concordances. Either we add a note to the existing PSMA source (my preference) or we add a new source.

**Comment by NickStallman** [comment]
Hey Guys

Just to be clear, PSMA ids are again different from the ABS ids. Just to make things annoying. :)
I believe PSMA ids are stable between data releases, and their releases are reasonably frequent.

Are separate datasources needed? I was thinking of just using ausstat and appending the year to the end of the concordance field for the ID.
My thinking is there will only ever be one set of ausstat sourced attributes. E.g. I don't think it would be needed or useful to keep the 2011 population numbers in WOF, instead they'd be replaced by the 2016 (or 2021) so they are always up to date.
In theory, the only thing WOF should have from 2011 is the old ID.
